core/src/main.cpp

```diff
@@ -288,6 +295,12 @@ constexpr auto default_jsonschema() -> std::string_view
                                     "required": [
                                         "user_attribute_mapping"
                                     ]
+                                }},
+                                {{
+                                    "required": [
+                                        "mapping_lib",
+                                        "mapping_config"
```

Comment:
need to match whatever is decided 30 lines above...

Reply:
Done.